gp2gp_adaptor_response_docs.yaml

```diff
@@ -0,0 +1,557 @@
+openapi: 3.0.1
+info:
+    title: OpenAPI definition
+    version: v0
+servers:
+    - url: http://localhost:8080/api
+      description: Generated server url
+paths:
+    /requests:
+        post:
+            tags:
+                - Ehr Status requests controller
+            operationId: getEhrRequestsEncodedForm
+            parameters:
+                - name: request
+                  in: query
+                  required: true
+                  schema:
+                      $ref: '#/components/schemas/EhrStatusRequestQuery'
```

Comment:
For my understanding, is this saying you can pass the filtering parameters in via query parameters? Also it's a bit strange it's marked as required, given that I don't think I've ever used the `/requests` endpoint in this way. 🤔